src/nvim/menu.c

```diff
@@ -679,7 +677,9 @@ static dict_T *menu_get_recursive(const vimmenu_T *menu, int modes)
   tv_dict_add_nr(dict, S_LEN("hidden"), menu_is_hidden(menu->dname));
 
   if (menu->mnemonic) {
-    mnemonic_len = utf_char2bytes(menu->mnemonic, (u_char *)buf);
+    int mnemonic_len;
+    char_u buf[8];  // > max value of char2bytes
```

Comment:
@jamessan Good idea but I guess that should be `char_u buf[8] = {'\0'};`

@justinmk #define MB_MAXCHARLEN 6 in mbyte.h ?